### ISSUE TITLE
Data Science with Python : Depression Detection using ML

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#3521]
Welcome to 'DSWP' Team, good to see you here

This issue will helps readers in gaining all the guidance that one needs to know about Depression Detection using ML. Tutorial to Depression Detection using ML and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Machine Learning

Mentors Assigned : AishanI Singh / Dhruv Bajaj

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by ron352** [comment]
Sarmistha Sarna 
Serial no:614
Type: documentation 
Approach: depression binary classification using various algorithm and accuracy check

**Comment by prathimacode-hub** [comment]
Issue assigned to @ron352 for documentation

**Comment by kishlayaug15** [comment]
Kishlay Kumar
Serial no : 501 
Type: documentation, Audio
Approach: Detection using Different ML and DL algorithms using Sklearn, CNN layers, TensorFlow and other required libraries

@prathimacode-hub  assign me this issue I'll work on it

**Comment by prathimacode-hub** [comment]
Issue is assigned to @kishlayaug15 for audio contribution

**Comment by kishlayaug15** [comment]
@prathimacode-hub I was working on audio of Depression Detection with ML #3521 & Image Retrieval #4386 . I have made necessary changes (added audio and description) for both and created a PR . Please check the PR #5588
